paper.md

```diff
@@ -2,11 +2,77 @@
 title: "Popper 2.0: A Container-Native Workflow Execution Engine For Testing Complex Applications and Reproducing Scientific Explorations"
 author: Ivo Jimenez, Jayjeet Chakraborty, Arshul Mansoori, Quincy Wofford and Carlos Maltzahn
 abstract: |
- Abstract
+ Software containers allow users to "bring their own environment" to 
```

Comment:
we can use the ABT framework to rewrite the abstract. This will be helpful as it will guide the structure of the introduction as well. In general, the abstract needs to answer: what's the problem? what's the current state of the art and why the problem is not solved? what are we doing to solve it?